docs/source/write_first_app.rst

```diff
@@ -997,7 +589,7 @@ Summary
 Now that we’ve seen how the sample application and chaincode are written and how they interact with each other, you should have a pretty
 good sense of how applications interact with a blockchain network using a smart
 contract to query or update the ledger. You’ve seen the basics of the roles
-smart contracts, APIs, and the SDK play in queries and updates and you should
+smart contracts, contract APIs, and application APIs play in queries and updates, and you should
```

Comment:
hmmm "contract" APIs immediately after smart contracts.

Reply:
They are not the same thing, like golf and golf clubs both contain "golf" but are not the same thing. The original text used "API" to mean "contract API" and "SDK" to mean "application API". They are both APIs. I'm not keen on the section in general though, so perhaps I can rework to make us both happier.